Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs_sys"
 description = "System crate for the Mozilla SpiderMonkey JavaScript engine."
 repository = "https://github.com/servo/mozjs/"
-version = "0.61.7"
+version = "0.62.0"
```

Comment:
I think previous updates used the same version number as SM (i.e. 0.66.0). Are you not going to keep using that versioning system?

Reply:
We gave up on that previously. The last update was 50-something iirc.

Reply:
The SM60 update went from 0.51.4 to 0.60.0.

Reply:
I was hoping to set the version to be the max of the previous version + 1 and the SM version, so it keeps in rough step with the SM version.